packages/web-components/src/anchor-button/anchor-button.spec.ts

```diff
@@ -103,4 +103,22 @@ test.describe('Anchor Button - Boolean Attributes', () => {
       await expect(element).toHaveJSProperty(attribute, value);
     });
   }
+
+  test(`should have transparent border when the \`disabled\` or \`disabled-focus\` attribute is present`, async ({
+    page,
+  }) => {
+    await element.evaluate((node: any) => {
+      node.getAttribute('appearance', 'primary');
```

Comment:
Did you mean to set this attribute?

```suggestion
      node.setAttribute('appearance', 'primary');
```

Reply:
I did mean that! Fixed.